### PR TITLE
Update names - Adobe Commerce

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
@voleye Can we update this to "Adobe Commerce" from "Magento"?

What about "Prerequisites? It says "Magento >= 2.4.x" Will this work on the new Adobe Commerce? If so, what version?
What version of PWA studio does it need?

**Comment by aluminium1989** [comment]
hi @rebekaburnett ,

Naming can be updated to Adobe Commerce.

As for versions, Adobe Commerce latest is still 2.4.* and modules works for it.
PWA Studio latest is 14.0.0. Vipps pwa module compatibility is not tested yet for this version.
I was developing on PWA Studio 13.* but most likely we're compatible with 14 as well, many dependencies weren't changed.

**Comment by rebekaburnett** [comment]
Thank you @aluminium1989 I will make an update and you can check. :-)

**Comment by rebekaburnett** [comment]
@aluminium1989 I don't know if the Vipps Payment Module version number is correct.
I'm not sure what that is - do you mean the [eCommerce API (2.0.3)](https://developer.vippsmobilepay.com/api/ecom/), the [ePayment API (1.4.1)
](https://developer.vippsmobilepay.com/api/epayment/), or something else maybe?

File: README.md [hunk 1]
```diff
@@ -24,14 +24,15 @@ END_METADATA -->
...
 
-This module supports adding Vipps single payment option to Magento.
+This module supports adding Vipps single payment option.
 
 ## Prerequisites
 
-* Magento >= 2.4.x
-* Vipps Payment Module >= 2.4.23
+* Adobe Commerce >= 2.4.x
+* PWA Studio latest >= 13.0.x.
```
Comment (aluminium1989) [inline README.md]: greater or equal 13.*

File: README.md [hunk 2]
```diff
@@ -24,14 +24,15 @@ END_METADATA -->
...
+* [Vipps/MobilePay Payment Module for Adobe Commerce](https://developer.vippsmobilepay.com/docs/plugins-ext/magento/) >= 2.4.23
```
Comment (aluminium1989) [inline README.md]: greater or equal 2.4 , same as line33

**Review by aluminium1989 — APPROVED** [review]

**Pull request merged**